### PR TITLE
core: --version option added

### PR DESCRIPTION
**Pull request by MathWay**

### CODE REVIEW

File: main.c [hunk 1]
```diff
@@ -39,6 +39,8 @@
 #include "utils.h"
 #include "worker.h"
 
+#define VERSION "dnsbalanser version 0.2.0"
```
Comment (pfactum) [inline main.c]: I'd suggest splitting this into 2 defines:

1. `DS_APP_NAME "dnsbalancer"`
2. `DS_APP_VERSION "0.2.0"`

Although, version by itself might be splitted into X.Y.Z parts, we won't do this ATM.

File: main.c [hunk 2]
```diff
@@ -104,6 +108,9 @@ int main(int _argc, char** _argv)
 				help();
 				stop_code(EX_OK, NULL);
 				break;
+			case 'g':
+				stop_code(EX_OK, VERSION);
```
Comment (pfactum) [inline main.c]: Then,

`stop_code(EX_OK, DS_APP_NAME ", v" DS_APP_VERSION);`

Comment (MathWay) [inline main.c]: Do we really need comma here? Like "dnsbalancer, v0.2.0"?

Comment (pfactum) [inline main.c]: Not necessarily.

File: main.c [hunk 3]
```diff
@@ -39,6 +39,9 @@
 #include "utils.h"
 #include "worker.h"
 
+#define DS_APP_NAME "dnsbalancer"
```
Comment (pfactum) [inline main.c]: May we have this properly indented with TABs please?

Comment (MathWay) [inline main.c]: Ok. Used the same spacing size as in pfcq.h

**Pull request merged**